src/field.cpp

```diff
@@ -913,6 +913,7 @@ bool map::process_fields_in_submap( submap *const current_submap,
                         }
                         // If the flames are in a brazier, they're fully contained,
                         // so skip consuming terrain
+                        item ash( "ash", calendar::turn );
```

Comment:
Field processing is performance critical, creating an `item` instance should only be done when you actually need it: not here, but right before the call to `map::add_item_or_charges`. You don't even need a separate instance, you can call the function with a temporary `add_item_or_charges( p, item( "ash" ) );` (`calendar::turn` is the default anyway).

Reply:
Fixed in fcd4864.